### ISSUE TITLE
New-VSStack throws exception

### ISSUE THREAD

**Issue by NikitaGor** [scrthq/VaporShell#52]
Hi Nate,

using latest version of VaporShell. New-VSStack function throws an exception

```powershell
ProcessRequest4 : Exception calling "DescribeStacks" with "1" argument(s): "Stack with id blah does not exist"
At \WindowsPowerShell\Modules\Vaporshell\2.7.1\VaporShell.psm1:273 char:13
+             ProcessRequest4 @PSBoundParameters
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [ProcessRequest4], MethodInvocationException
    + FullyQualifiedErrorId : AmazonCloudFormationException,ProcessRequest4
```

I saw that you've put in a note that it was fixed. However I'm still seeing it.
`New-VSStack -TemplateBody $content -StackName $StackName -Capabilities $Capabilities -Parameters $awsParams -Tags $stackTags -Confirm:$false -Verbose`

Can you please check that one?

Thank you

**Comment by scrthq** [comment]
Hey @NikitaGor - On it! I added in additional functionality to check if a stack exists already and create a Change Set if so, it looks like it's failing unexpectedly though when the stack does not exist. I should have this fixed by tonight and published to the Gallery! Thanks for opening this up!

**Comment by scrthq** [comment]
@NikitaGor - When you're running this, are you setting `$ErrorActionPreference = 'Stop'` anywhere? Trying to replicate now.

**Comment by NikitaGor** [comment]
@scrthq nope

**Comment by scrthq** [comment]
@NikitaGor can you run that `New-VSStack` call with `-Verbose` added and provide the Verbose output?

**Comment by scrthq** [comment]
For context, the issue that was fixed in `New-VSStack` in the last version was caused when a stack did actually already exist. I'm not able to replicate the issue you have currently with the latest version in the gallery (2.7.1) with a truly new stack on my end, so it's a little perplexing. Verbose output should tell me a bit more about where things are hanging and if it's hitting a loop anywhere and failing after that.

**Comment by NikitaGor** [comment]
Hi Nate, sorry for taking a while to get back to you.
Turns out there's some weird stuff happening on our end. We wrote a wrapper around VS that constructs our CF templates and on top of that wrapper we built a custom DSL to declare a stack. That DSL is using classes and methods in a OO way. When calling a wrapper function from our DSL we started seeing exceptions, but we've put a fix around that issue. So it's all working fine now.

Thanks!
